ntp-udp/src/raw_socket.rs

```diff
@@ -61,7 +61,7 @@ mod set_timestamping_options {
 
     fn configure_timestamping_socket(
         udp_socket: &std::net::UdpSocket,
-        method: TimestampMethod,
```

Comment:
why remove the type safety here? this was meant as a safeguard to just passing in integers. An extra method can be added to the enum if that would help.

Reply:
Well, because it's not *a timestamp method* (but rather an extra configuration sockopt for an existing one) and it shouldn't be exposed outside of this module… I guess the better option would be like:

```rust
enum SocketOption {
    Method(TimestampMethod),
    #[cfg(target_os = "freebsd")]
    TimestampClock,
}
```